source/index.rst

```diff
@@ -23,6 +23,14 @@ you see something wrong, something that could be better, or want to
 contribute examples or descriptions, feel free to use the buttons at
 the top right to file an issue on GitHub or suggest an edit.
 
+A basic SYCL program, taken from the SYCL 2020 specification, looks like this:
+
+[source,,linenums]
+----
+include::anatomy.cpp[lines=4..-1]
+----
```

Comment:
This is Asciidoc syntax, which causes a build error because these docs are using reStructuredText (rst).  I don't know this documentation tool well, but you can see an example of how to add a code snippet in "source/iface/device-selector.rst":

```
.. literalinclude:: /examples/gpu-selector.cpp
   :lines: 5-
   :linenos:
```

I think you want something like that.

Reply:
Thanks @gmlueck - I'm struggling to get local builds working on my machine. Do you know if there is a straight forward way to get all the plugins installed locally?

Reply:
It was a while ago, but I think I just installed Python 3 and then:

```
$ pip install -r requirements.txt
```

Reply:
Then to do a build:

```
$ make html
```

The output is in "build/html"